main.js

```diff
@@ -1,3 +1,3 @@
-import mapContentToTopper from './src/js/map-content-to-topper.js';
+import mapContentToTopper from '@financial-times/n-map-content-to-topper';
```

Comment:
This needs to be written using the bower package name which is not namespaced.

When it gets published to npm, Origami uses a [tool](https://github.com/Financial-Times/origami-component-converter) to convert the imports into their npm equivalent. 

Sorry for all these weird quirks, when/if Origami stops supporting Bower, then writing Origami components which use dependencies will be much simpler.

```suggestion
import mapContentToTopper from 'n-map-content-to-topper';
```

Reply:
Great, will fix now :)

Reply:
And no need to be sorry! I've learned stuff in the process.